### PR TITLE
Widget UI improvements

### PR DESCRIPTION
**Pull request by operramon**

### CODE REVIEW

File: libs/search-widget/src/components/modal/Modal.svelte [hunk 1]
```diff
@@ -100,11 +100,25 @@
...
 
+  .modal.transparent .modal-content {
+    overflow: auto;
+    padding-right: 16px;
+  }
+  .modal.transparent .modal-content::-webkit-scrollbar {
+    width: 6px;
+  }
+  .modal.transparent .modal-content::-webkit-scrollbar-thumb {
+    background-color: #000;
```
Comment (ebrehault) [inline libs/search-widget/src/components/modal/Modal.svelte:116]: use a `var` (define new ones in `wdiget.svelte` if this color is not existing yet)

File: libs/search-widget/src/components/modal/Modal.svelte [hunk 2]
```diff
@@ -100,11 +100,25 @@
...
+  }
+  .modal.transparent .modal-content::-webkit-scrollbar-track {
+    background-color: #fff;
```
Comment (ebrehault) [inline libs/search-widget/src/components/modal/Modal.svelte:119]: use a `var` (define new ones in `wdiget.svelte` if this color is not existing yet)

File: libs/search-widget/src/widgets/row/Row.svelte [hunk 3]
```diff
@@ -65,19 +65,25 @@
...
   }
   .title {
+    margin-bottom: 0.4em;
     font-weight: var(--font-weight-bold);
   }
+  .byline {
+    font-size: 12px;
+    font-style: italic;
+    padding-bottom: 10px;
+    border-bottom: 1px solid #000;
```
Comment (ebrehault) [inline libs/search-widget/src/widgets/row/Row.svelte:85]: use a `var` (define new ones in `wdiget.svelte` if this color is not existing yet)

File: libs/search-widget/src/widgets/suggestions/Suggestions.svelte [hunk 4]
```diff
@@ -60,5 +60,6 @@
     text-align: right;
     font-size: small;
     margin: 0;
+    color: #707070;
```
Comment (ebrehault) [inline libs/search-widget/src/widgets/suggestions/Suggestions.svelte:63]: use a `var` (define new ones in `wdiget.svelte` if this color is not existing yet)

**Review by ebrehault — APPROVED** [review]

File: libs/search-widget/src/components/modal/Modal.svelte (hunk 1 above)
Comment (operramon) [reply libs/search-widget/src/components/modal/Modal.svelte]: fixed

File: libs/search-widget/src/components/modal/Modal.svelte (hunk 2 above)
Comment (operramon) [reply libs/search-widget/src/components/modal/Modal.svelte]: fixed

File: libs/search-widget/src/widgets/row/Row.svelte (hunk 3 above)
Comment (operramon) [reply libs/search-widget/src/widgets/row/Row.svelte]: fixed

**Pull request merged**